docs/upgrading-fcos.md

```diff
@@ -21,14 +21,105 @@ There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` con
 
 The following is a list of notable new features.
 
+### Offline Tang provisioning
+
+The `tang` section gained a new `advertisement` field. If specified, Ignition will use it to provision the Tang server binding rather than fetching the advertisement from the server at runtime. This allows the server to be unavailable at provisioning time. The advertisement can be obtained from the server with `curl http://tang.example.com/adv`.
```

Comment:
We'll also need a mention of `boot_device.luks.tang.advertisement`, and it should be included in the example.

Reply:
Gave it a swing!